xml/XSLT/throttle-config.xsl

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Stylesheet to convert a JMRI throttle XML file into displayable HTML -->
+<!-- Used by default when the throttle file is displayed in a web browser-->
+<!-- This is just a basic implementation for debugging purposes, without -->
+<!-- any real attempt at formatting -->
+<!-- This file is part of JMRI.  Copyright 2007-2022.                       -->
+<!--                                                                        -->
+<!-- JMRI is free software; you can redistribute it and/or modify it under  -->
+<!-- the terms of version 2 of the GNU General Public License as published  -->
+<!-- by the Free Software Foundation. See the "COPYING" file for a copy     -->
+<!-- of this license.                                                       -->
+<!--                                                                        -->
+<!-- JMRI is distributed in the hope that it will be useful, but WITHOUT    -->
+<!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
+<!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
+<!-- for more details.                                                      -->
+
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+  <!-- Need to instruct the XSLT processor to use HTML output rules.
+     See http://www.w3.org/TR/xslt#output for more details
+-->
+
+  <xsl:output method="html" encoding="ISO-8859-1"/>
```

Comment:
Do we want to have ISO-8859-1 encoding? UTF-8 seems to be a better choice.

Reply:
This is what was in throttle.xsl that I based my work on.  I will rely on others to make the decision.

Reply:
@danielb987 @dsand47 Still an open question...

Reply:
`utf-8` is probably the best choice.  About 1/3 of our existing XSLT transforms use it. User input to Java is natively in UTF, so being able to display that is good.

Reply:
I agree with @bobjacobsen. ISO-8859-1 works fine with Western European languages but might cause problems with other languages. The problem with ISO-8859-1 is that it only has 256 characters so it's not able to display all possible characters.

Reply:
Changed to UTF-8